dataset/dictionary/characters.json5

```diff
@@ -3312,6 +3324,8 @@
   {
     en: "Melusine",
     ja: "メリュジーヌ",
+    zhCN: "美露莘",
+    pinyins: [{ char: "莘", pron: "xin3" }],
```

Comment:
The pronunciation of this character should be shen1 or xin1, as xin1 is used in names of locations, I think it should be pronounced as shen1

Ref:
![image](https://github.com/xicri/genshin-langdata/assets/48173350/2b94f250-16bc-4049-8b7f-69e64a918dad)

Reply:
I also made a mistake. This character here is just used as the transliteration of the original English text, so xin1 is correct.

Reply:
@Bill-Haku OK, I understand xin1 is correct.